odoo/addons/base/models/ir_qweb.py

```diff
@@ -2548,8 +2550,8 @@ def _get_cached_values(self, cache_key, get_value):
         'xml' not in tools.config['dev_mode'],
         tools.ormcache('bundle', 'css', 'js', 'tuple(sorted(assets_params.items()))', 'rtl', cache='assets'),
     )
-    def _generate_asset_links_cache(self, bundle, css=True, js=True, assets_params=None, rtl=False):
-        return self._generate_asset_links(bundle, css, js, False, assets_params, rtl)
+    def _generate_asset_links_cache(self, bundle, css=True, js=True, assets_params=None, rtl=False, autoprefix=False):
```

Comment:
Looks like you forgot to add `autoprefix` to the ormcache

Didn't test it but you will have an issue if the same assetbundle is both use with autoprefix and without autoprefix. The last value in cache will be used. Also, I wonder why it is a parameter from the t-call-assets and not from the bundle itself. Does it make sense to have the same assetbundle called with and without the autoprefix?